### ISSUE TITLE
React

### ISSUE THREAD

**Issue by sarahgjacobs** [sarahgjacobs/tuter#4]
@thebobfather2 Would you be interesting in heading this since it's an area you're more familiar with?

**Comment by thebobfather2** [comment]
@sarahgjacobs absolutely! i might need to restructure it a bit, i want to look into how it will be deployed tomorrow before i mess with it though
